msg/UAVCommand.msg

```diff
@@ -0,0 +1,12 @@
+builtin_interfaces/Time time_stamp
+string sender_id
+uint8 type
+#enum
```

Comment:
Könntest du die enums nach ganz unten verschieben, damit sich die beiden nicht mischen?

Reply:
Sry hab ich net gesehen, ist wie in der Politik. Kommt noch, ganz sicher!